ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -1,313 +1,333 @@
-            }
-        },
-        "virtualNetworkResourceGroupName": {
-            "type": "string",
-            "metadata": {
-                "description": "The resource group containing the existing virtual network."
-            }
-        },
-        "createNetworkSecurityGroup": {
-            "type": "bool",
-            "metadata": {
-                "description": "Whether to create a new network security group or use an existing one"
-            },
-            "defaultValue": false
-        },
-        "networkSecurityGroupId": {
-            "type": "string",
-            "metadata": {
-                "description": "The resource id of an existing network security group"
-            },
-            "defaultValue": ""
-        },
-        "networkSecurityGroupRules": {
-            "type": "array",
-            "metadata": {
-                "description": "The rules to be given to the new network security group"
-            },
-            "defaultValue": []
-        },
-        "availabilitySetTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the availability set"
-            },
-            "defaultValue": {}
-        },
-        "networkInterfaceTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the network interfaces"
-            },
-            "defaultValue": {}
-        },
-        "networkSecurityGroupTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the network security groups"
-            },
-            "defaultValue": {}
-        },
-        "virtualMachineTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the virtual machines"
-            },
-            "defaultValue": {}
-        },
-        "imageTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the images"
-            },
-            "defaultValue": {}
-        },
-        "deploymentId": {
-            "type": "string",
-            "metadata": {
-                "description": "GUID for the deployment"
-            },
-            "defaultValue": ""
-        },
-        "apiVersion": {
-            "type": "string",
-            "metadata": {
-                "description": "WVD api version"
-            },
-            "defaultValue": "2019-12-10-preview"
-        },
-        "ouPath": {
-            "type": "string",
-            "metadata": {
-                "description": "OUPath for the domain join"
-            },
-            "defaultValue": ""
-        },
-        "domain": {
-            "type": "string",
-            "metadata": {
-                "description": "Domain to join"
-            },
-            "defaultValue": ""
-        }
+  "parameters": {

```

Comment:
nit: should we use 4 spaces for indentation?

Reply:
fix spacing

Reply:
create has 2 spaces for tabs, add has 4 spaces.  should i make them consistent at 4 spaces?